plugins/woocommerce/includes/wc-cart-functions.php

```diff
@@ -541,3 +541,55 @@ function wc_get_cart_item_data_hash( $product ) {
 		)
 	);
 }
+
+
+if (!function_exists('wc_get_cart_item_data')) {
+	/**
+	 * Get cart item data
+	 * 
+	 * @param $cart_item
+	 * @return array 
+	 */
+    function wc_get_cart_item_data($cart_item) {
+        $item_data = array();
+
+        $is_taxonomy = false;
+        $taxonomy_slug = null;
+        $term_slug = null;
+
+        // Variation values are shown only if they are not found in the title as of 3.0.
+        // This is because variation titles display the attributes.
+        if ( $cart_item['data']->is_type( 'variation' ) && is_array( $cart_item['variation'] ) ) {
```

Comment:
It seems like we only populate `$item_data` for variations and so, for non-variations, we'll just return `[]`. Is that intentional? Can you speak a little more to this, or add appropriate notes in the function doc block?